pyat/at/physics/rdt.py

```diff
@@ -182,6 +183,36 @@ def pf(i, j):
             b4lm * betaym * betaym / 64 * pym * pym * pym * pym * pf(0, 4)
         )
 
+    if (RDTType.CHROMATIC2 in rdttype) or (RDTType.ALL in rdttype):
+        b4lm = b4l[mask_b4l]
+        betaxm = betax[mask_b4l]
+        betaym = betay[mask_b4l]
+        etaxm = etax[mask_b4l]
+        pxm = px[mask_b4l]
+        pym = py[mask_b4l]
+        rbetaxm = np.sqrt(betaxm)
+        etaxm2 = etaxm * etaxm
+        cpym = np.conj(pym)
+        rdts["h21001"] = sum(3.0 / 8 * b4lm * etaxm * rbetaxm * betaxm * pxm * pf(1, 0))
+        rdts["h30001"] = sum(
+            1.0 / 8 * b4lm * etaxm * rbetaxm * betaxm * pxm * pxm * pxm * pf(3, 0)
```

Comment:
Would not it be more efficient to take the constant 3.0 / 8, or 1.0 / 8 and others out of the summation ?
```python
3.0 / 8 * sum(...)
```

Reply:
And same for `pf(...)`